src/pages/lists/action-list/action-list-local-actions/device-edit-mobile.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import { createStyles, makeStyles } from '@material-ui/core';
+import Divider from '@material-ui/core/Divider';
+
+type DeviceEditProps = {
+    subTitle: string;
+    updateSubTitle: (tempSubTitle: string) => void;
+    navigateBack: () => void;
+};
+
+const useStyles = makeStyles(() =>
+    createStyles({
+        dialogDivider: {
+            width: '100%',
+        },
+        textField: {
+            margin: '16px',
+            width: 'calc(100% - 32px)',
+        },
+        button: {
+            margin: '16px',
+            width: 'calc(100% - 32px)',
+        },
+    })
+);
+
+export const DeviceEditMobile = (props: DeviceEditProps): JSX.Element => {
```

Comment:
This implementation picked up white200 as its background color, thus making the text field shading invisible

![image](https://user-images.githubusercontent.com/8997218/145494190-4cc7ac6c-5992-41be-b6c6-f4deabbcad57.png)